src/v/cluster/partition_probe.cc

```diff
@@ -167,6 +167,35 @@ void replicated_partition_probe::setup_internal_metrics(const model::ntp& ntp) {
       {},
       {sm::shard_label, partition_label});
 
+    if (model::is_user_topic(_partition.ntp())) {
+        _metrics.add_group(
+          cluster_metrics_name,
+          {
+            sm::make_gauge(
+              "iceberg_offsets_pending_translation",
+              [this] {
+                  return _partition.log()->config().iceberg_enabled()
+                           ? _iceberg_translation_offset_lag
+                           : 0;
+              },
+              sm::description("Total number of offsets that are pending "
+                              "translation to iceberg."),
+              labels),
+            sm::make_gauge(
+              "iceberg_offsets_pending_commit",
+              [this] {
+                  return _partition.log()->config().iceberg_enabled()
+                           ? _iceberg_commit_offset_lag
+                           : 0;
```

Comment:
Should we emit -1 if iceberg is disabled so that we can distinguish between not on and up to date?

Reply:
thats a good idea.

Reply:
If we default to `-1` for not initialized, then maybe we emit `-2` or something for "not enabled". I don't know just spit balling.

Reply:
ya that makes sense to me, done.